ios/RCCViewController.m

```diff
@@ -469,6 +469,54 @@ -(void)setStyleOnAppearForViewController:(UIViewController*)viewController appea
   } else {
     self.navBarHairlineImageView.hidden = NO;
   }
+  
+  NSString *navBarCustomView = self.navigatorStyle[@"navBarCustomView"];
+  if (navBarCustomView) {
+    NSLog(@"navBarCustomView:%@", navBarCustomView);
+    if ([self.view isKindOfClass:[RCTRootView class]]) {
```

Comment:
Defensive coding :), it's better practice to check this is the right object before `casting`.